lib/drivers/redis.js

```diff
@@ -110,6 +110,9 @@
                 data[data.length - 1] = data[data.length - 1].map(function (index) {
                     return parseInt(index, 10);
                 });
+                data.forEach(function (row, index) {
```

Comment:
I'm unclear on whether it's possible for redis to return the results out of order and have this come back in a weird way. I've not seen it happen in testing, but hey, that's testing.